### PR TITLE
Doubling throuput

### PR DESCRIPTION
**Pull request by procrastinatio**

### CODE REVIEW

File: dump.py [hunk 1]
```diff
@@ -22,8 +22,9 @@
 DUMP_READ_THROUGHPUT = 60
 DUMP_WRITE_THROUGHPUT = 5
```
Comment (loicgasser) [inline dump.py]: DUMP_WRITE_THROUGHPUT = 5 (let's leave it at 10)

not related to this PR

File: dump.py [hunk 2]
```diff
@@ -22,8 +22,9 @@
 DUMP_READ_THROUGHPUT = 60
```
Comment (loicgasser) [inline dump.py]: should be 40 no?
Otherwise we need to do it in 2 steps

**Comment by gjn** [comment]
This is only for the dump? Or general dynamodb setting also for API?

**Comment by loicgasser** [comment]
for dump we increase read capacity and reduce it when done.

the baseline for API is BASE_READ_THROUGHPUT

File: dump.py (hunk 2 above)
Comment (procrastinatio) [inline dump.py]: Did not change anything...

**Comment by procrastinatio** [comment]
Yes we increase the thoughput capacity before starting the dumps, and decreasing once done.

**Comment by loicgasser** [comment]
You can only increase by doubling it.
You changed the code here:
https://github.com/geoadmin/dynamodump/commit/0613fe057023ad3acebe65aff18210976ba94d38

so it currently fails to increase the read units I believe

**Comment by gjn** [comment]
How do we deploy this? Or not yet?

**Comment by procrastinatio** [comment]
There is nothing to deploy, we have only two live dynamodb tables. 

@loicgasser I did increase yesterday read throuhput manually from 10 to 60, without any trouble, and today it has its baseline again. But I a monitoring the throuput capacity changes throuput through time to further check.

**Pull request merged**